editor/plugins/packed_scene_translation_parser_plugin.cpp

```diff
@@ -50,21 +51,31 @@ Error PackedSceneEditorTranslationParserPlugin::parse_file(const String &p_path,
 	Ref<SceneState> state = Ref<PackedScene>(loaded_res)->get_state();
 
 	Vector<String> parsed_strings;
-	String property_name;
-	Variant property_value;
 	for (int i = 0; i < state->get_node_count(); i++) {
-		if (!ClassDB::is_parent_class(state->get_node_type(i), "Control") && !ClassDB::is_parent_class(state->get_node_type(i), "Viewport")) {
+		String node_type = state->get_node_type(i);
+		if (!ClassDB::is_parent_class(node_type, "Control") && !ClassDB::is_parent_class(node_type, "Viewport")) {
```

Comment:
This was in the original code, but why does it check Viewport? It should be Window, no?